crates/cli/src/subcommands/call.rs

```diff
@@ -19,7 +20,7 @@ pub fn cli() -> clap::Command {
         .arg(
             Arg::new("database")
                 .required(true)
-                .help("The database domain or address to use to invoke the call"),
+                .help("The database name or identity to use to invoke the call"),
```

Comment:
Note for @bfops we should probably do some code deduplication on this param because it's used quite frequently it seems.